pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -286,14 +271,6 @@ def _check_config_and_set_final_flags(
         if accelerator is not None:
             if accelerator in self._accelerator_types or accelerator == "auto" or isinstance(accelerator, Accelerator):
                 self._accelerator_flag = accelerator
-            elif accelerator in self._registered_strategies or isinstance(accelerator, Strategy):
```

Comment:
Should we add a else to throw an exception here if user passed in an unrecognizable accelerator?

Reply:
yes and no. That would need more work and I think outside of scope here. see #12698